### PR TITLE
Preparing release 1.5.0

### PR DESCRIPTION
**Pull request by Natim**
**Step 1**

```
$ git co -b prepare-X.X
$ prerelease
$ vim docs/conf.py
$ git ci -a --amend
$ git push origin prepare-X.X
```
- [x] Merge remaining pull requests
- [x] Update changelog
- [x] Update version in docs/conf.py
- [x] Version dependencies
- [x] If cliquet was updated, update the link to default settings in the docs 

**Step 2**

```
$ git co master
$ git merge --no-ff prepare-X.X
$ release
$ postrelease
```
- [x] Tag vX.Y.Z
- [x] Publish on Pypi
- [x] Revert version pinning

**Step 3**
- [x] Close milestone in Github
- [x] Add entry in Github release page
- [x] Create next milestone in Github
- [x] Configure the version in ReadTheDocs
- [x] Send mail to ML

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,10 +4,21 @@ CHANGELOG
...
 ------------------
 
-- Nothing changed yet.
+**Protocol**
+
+- Make sure batch always return 200 except for 5xx errors. (#78)
+
+**Bug fixes**
+
+- Fix If-None-Match header format (#76)
```
Comment (leplatrem) [inline CHANGELOG.rst]: nit1: `If-None-Match`
nit2: Fix what format?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,10 +4,21 @@ CHANGELOG
...
+
+**Internal changes**
+
+- Add a Dockerfile (#77)
+- Remode documentation warnings (#74)
```
Comment (leplatrem) [inline CHANGELOG.rst]: typo: Remove

File: dev-requirements.txt [hunk 3]
```diff
@@ -7,4 +7,3 @@ requests
 nose
 nose-cov
 nose-mocha-reporter
-https://github.com/mozilla-services/cliquet/archive/master.zip
```
Comment (leplatrem) [inline dev-requirements.txt:10]: You don't add a travis rule for cliquet master?

Comment (Natim) [inline dev-requirements.txt:10]: I definitely should as we did for Kinto :+1:

**Pull request merged**